modules/t/seqRegionUtils.t

```diff
@@ -0,0 +1,45 @@
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+use strict;
+use warnings;
+
+use Test::More;
+use Bio::EnsEMBL::Test::MultiTestDB;
+
+BEGIN {
+    use_ok('Bio::EnsEMBL::Variation::Utils::SeqRegionUtils', qw(update_seq_region_ids));
+}
+
+my $multi = Bio::EnsEMBL::Test::MultiTestDB->new('homo_sapiens');
+my $cdb = $multi->get_DBAdaptor('core');
+my $vdb = $multi->get_DBAdaptor('variation');
+
+## check no seq_region update performed when last is not related to current release
```

Comment:
Suggest comment expand on what last is.
Also might be useful to note that test databases do not have release and assembly on dbname.